mintpy/closure_phase_bias.py

```diff
@@ -8,39 +8,106 @@
 
 import os
 import sys
+import time
 import argparse
 import numpy as np
+import glob
+from datetime import datetime as dt
 
-from mintpy.objects import ifgramStack
-from mintpy.utils import readfile, writefile
+from mintpy.objects import ifgramStack, cluster
+from mintpy.utils import readfile, writefile, ptime, isce_utils, arg_group
 from mintpy import ifgram_inversion as ifginv
 
-
 ################################################################################
 REFERENCE = """reference:
-  Zheng, Y., et al., (2022) On Closure Phase and Systematic Bias in Multilooked SAR Interferometry, IEEE TGRS, under review (minor revision)
+  Y. Zheng, H. Fattahi, P. Agram, M. Simons and P. Rosen, "On Closure Phase and Systematic Bias in Multi-looked SAR Interferometry," in IEEE Transactions on Geoscience and Remote Sensing, doi: 10.1109/TGRS.2022.3167648.
 """
 EXAMPLE = """example:
-  closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --numsigma 2.5
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --action create_mask
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --numsigma 2.5 --action create_mask
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --bw 10 -a quick_biasEstimate
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --bw 10 -a biasEstimate
 """
 
 def create_parser():
-    parser = argparse.ArgumentParser(description = 'Create an indication map for closure phase bias.')
+    parser = argparse.ArgumentParser(description = 'This script deals with phase non-closure related bias, either in terms of masking, estimation or correction.',
+                                    formatter_class = argparse.RawTextHelpFormatter,
+                                    epilog=REFERENCE+'\n'+EXAMPLE)
     parser.add_argument('-i','--ifgramstack',type = str, dest = 'ifgram_stack',help = 'interferogram stack file that contains the unwrapped phases')
     parser.add_argument('--nl', dest = 'nl', type = int, default = 20, help = 'connection level that we are correcting to (or consider as no bias)')
+    parser.add_argument('--bw', dest = 'bw', type = int, default = 10, help = 'bandwidth of time-series analysis that you want to correct')
     parser.add_argument('--numsigma',dest = 'numsigma', type = float, default = 3, help = 'Threashold for phase (number of sigmas,0-infty), default to be 3 sigma of a Gaussian distribution (assumed distribution for the cumulative closure phase) with sigma = pi/sqrt(3*num_cp)')
```

Comment:
```suggestion
    parser.add_argument('--num-sigma',dest = 'num_sigma', type = float, default = 3, help = 'Threashold for phase (number of sigmas,0-infty), default to be 3 sigma of a Gaussian distribution (assumed distribution for the cumulative closure phase) with sigma = pi/sqrt(3*num_cp)')
```
To help the readability.

Reply:
Okay replaced with num_sigma